src/test/java/no/sikt/sws/IndexHandlerTest.java

```diff
@@ -177,12 +182,26 @@ void shouldHandleSearchRequestWithQueryParameters() throws IOException {
                 .stripper(WORKSPACEPREFIX)
                 .toJsonCompact();
 
-        assertEquals(compact(testcase.getResponseStripped()), compact(resultBody));
+        assertEquals(compact(testCase.getResponseStripped()), compact(resultBody));
 
         assertThat(response.getStatusCode(), is(equalTo(HTTP_OK)));
     }
 
     private String compact(String body) {
         return body.replaceAll("[\n\r ]", EMPTY_STRING);
     }
+
+    static Stream<TestCaseSws> allRequestArguments() {
+        var streamBuilder = Stream.<TestCaseSws>builder();
+
+        loadTestCases(streamBuilder, "proxy/requests-search.json");
+
+        return streamBuilder.build();
+    }
+
+    private static void loadTestCases(Stream.Builder<TestCaseSws> streamBuilder, String filename) {
```

Comment:
I see that you copied this from PrefixStripperTest - maybe we should just move it to TestCaseSws or TestUtils?